### ISSUE TITLE
Surveillance section: add coming soon

### ISSUE THREAD

**Issue by rskikuli** [Debatikcenter/debatikcenter#4]
At the surveillance section please add this image https://unsplash.com/photos/Q1rkSa4poHM.
@Pleurad canyou add a 'soon text to the image and add (upload) it here?

@Pleurad if you have a photo of a surveillance camera from Tirana even better :)

**Comment by debatik** [comment]
We are working for the Surveillance photo . For the moment is fine like this. If you have ideas please suggest.

**Comment by rskikuli** [comment]
@silvaarapi please use the image mentioned above for the moment.

**Comment by sidorelauku** [comment]
Page created with the image provided from @rskikuli.
Here: http://debatikcenter.net/surveillance.html

**Comment by ArmandoLulaj** [comment]
We are working for the Surveillance photo. 
We have an idea for a more general view on Surveillance concept. For the moment it is possible to remove this photo? A white of black page would do more for now. Redon was telling us that its from Greece and alike you we would rather prefer an authentic one.

**Comment by sidorelauku** [comment]
Image is removed now it's a white page.

**Comment by ArmandoLulaj** [comment]
Great. Thank you

**Comment by rskikuli** [comment]
Closing this since it is now at the stage that it is requested. Will open a new issue once the material is ready.
